server/mailet/integration-testing/src/test/java/org/apache/james/mailets/SubAddressingTest.java

```diff
@@ -113,90 +115,21 @@ void subAddressedEmailShouldBeDeliveredInINBOXWhenNobodyHasRight(@TempDir File t
         assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("OK GETACL");
 
         // do not give posting rights
-        testIMAPClient.sendCommand("SETACL " + TARGETED_MAILBOX + " " + "anybody" + " -p");
+        testIMAPClient.sendCommand("SETACL " + TARGETED_MAILBOX + " " + "anyone" + " -p");
         assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("\"any\" \"owner\"");
 
         sendSubAddressedMail();
         awaitSubAddressedMail(MailboxConstants.INBOX);
     }
 
     @Test
-    void subAddressedEmailShouldBeDeliveredInSpecifiedFolderWhenAnybodyHasRight(@TempDir File temporaryFolder) throws Exception {
+    void subAddressedEmailShouldBeDeliveredInSpecifiedFolderWhenAnyoneHasRight(@TempDir File temporaryFolder) throws Exception {
         setup(temporaryFolder);
 
-        // create mailbox
-        testIMAPClient.sendCommand("CREATE " + TARGETED_MAILBOX);
-        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("OK GETACL");
-
-        // give posting rights for anybody
-        testIMAPClient.sendCommand("SETACL " + TARGETED_MAILBOX + " " + "anybody" + " +p");
-        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("\"anybody\" \"p\"");
-
-        sendSubAddressedMail();
-        awaitSubAddressedMail(TARGETED_MAILBOX);
-    }
-
-    @Test
-    void subAddressedEmailShouldBeDeliveredInSpecifiedFolderWhenSenderIsWhitelisted(@TempDir File temporaryFolder) throws Exception {
-        setup(temporaryFolder);
-
-        // create mailbox
-        testIMAPClient.sendCommand("CREATE " + TARGETED_MAILBOX);
-        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("OK GETACL");
-
-        // whitelist sender 1 and send from sender 1
-        testIMAPClient.sendCommand("SETACL " + TARGETED_MAILBOX + " " + FROM + " +p");
-        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("\"user@james.org\" \"p\"");
-
-        sendSubAddressedMail();
-        awaitSubAddressedMail(TARGETED_MAILBOX);
-    }
-
-    @Test
-    void subAddressedEmailShouldBeDeliveredInINBOXWhenSenderIsNotWhitelisted(@TempDir File temporaryFolder) throws Exception {
-        setup(temporaryFolder);
-
-        // create mailbox
         testIMAPClient.sendCommand("CREATE " + TARGETED_MAILBOX);
-        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("OK GETACL");
-
-        // whitelist sender 2 and send from sender 1
-        testIMAPClient.sendCommand("SETACL " + TARGETED_MAILBOX + " " + FROM2 + " +p");
-        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("\"user4@james.org\" \"p\"");
-
-        sendSubAddressedMail();
-        awaitSubAddressedMail(MailboxConstants.INBOX);
-    }
-
-    @Test
-    void subAddressedEmailShouldBeDeliveredInINBOXWhenSenderIsBlacklisted(@TempDir File temporaryFolder) throws Exception {
```

Comment:
Why removing all those tests?

Reply:
you said that they had not much to do with subaddressing and shall be done in a dedicated location, so I thought those weren't needed anymore ?

Reply:
No, the intermediate assertions have nothing to do with SubAdressing but the overall test looks ok.